### PR TITLE
Ionicons

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@gvn R>

File: components/inlineEditor/index.less [hunk 1]
```diff
@@ -31,9 +31,10 @@
             border: 1px solid rgba(50, 178, 210, 0.5);
             background-color: rgba(225, 245, 250, 0.5);
             color: rgb(50, 178, 210);
-            font-size: 20px;
             width: 35px;
             height: 35px;
+            padding: 0;
+            //font-size: 13px;
```
Comment (gvn) [inline components/inlineEditor/index.less]: Remove comment

File: static/styles/common.less [hunk 2]
```diff
@@ -1,9 +1,13 @@
-@import (less) "../../node_modules/font-awesome/css/font-awesome.css";
+@import (less) "../../node_modules/webmaker-app-icons/css/ionicons.css";
 @import "normalize";
 @import "variables";
 @import "mixins";
 @import "fonts";
 
+// .ion {
```
Comment (gvn) [inline static/styles/common.less]: remove commented lines

**Comment by gvn** [comment]
R+ w nits

**Pull request merged**